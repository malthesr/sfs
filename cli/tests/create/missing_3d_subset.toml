args = [
  "create",
  "-s",
  "sample0=group0,sample1=group1,sample4=group4",
  "tests/create/missing.bcf"
]
