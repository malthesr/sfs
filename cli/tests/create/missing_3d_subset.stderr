[sfs  warn] Skipping record at position 'chr1:1' due to missing genotype. This error will be shown only once, with a summary at the end.
[sfs  warn] Skipped 2 records due to missing genotype.
